### PR TITLE
style(docs): MFLT-11822 - Update arrows styling 

### PR DESCRIPTION
**Pull request by charlesvestal**
### Summary

To provide more consistent, semantically sound, and beautiful documentation, arrows have been updated from `->` or `>` to `→`, used mostly when referencing a menu hierarchy.  I also considered modifying our linter to replace them, but did not want to collide with other functional uses in comments or code examples, etc.

### Internal Documentation

Aligns with https://github.com/memfault/memfault/pull/11178

### Test Plan

1. Run docs locally
2. Observe changed pages
3. See glorious arrows

### CODE REVIEW

File: static/swagger/memfault-api.json
Comment (charlesvestal) [inline static/swagger/memfault-api.json:1]: I'd love an eye on this one, in case it's something that shouldn't be manually updated.

Comment (tyhoff) [reply static/swagger/memfault-api.json:1]: I would probably back it out. We also don't use it atm so it's not worth updating.

**Review by tyhoff — DISMISSED** [review]
Love this! 🚢

File: static/swagger/memfault-api.json
Comment (charlesvestal) [reply static/swagger/memfault-api.json:1]: Done, thank you!

**Pull request merged**